hypervisor/arch/x86/guest/vmcs.c

```diff
@@ -19,6 +19,24 @@
 #include <asm/guest/vmexit.h>
 #include <logmsg.h>
 
+void clear_vmcs_bit(uint32_t vmcs_field, uint32_t bit)
+{
+	uint64_t val64;
+
+	val64 = exec_vmread(vmcs_field);
+	val64 &= ~bit;
+	exec_vmwrite(vmcs_field, val64);
+}
+
+void set_vmcs_bit(uint32_t vmcs_field, uint32_t bit)
+{
+	uint64_t val64;
+
+	val64 = exec_vmread(vmcs_field);
+	val64 |= bit;
+	exec_vmwrite(vmcs_field, val64);
+}
```

Comment:
These 2 APIs can be inline API with single line. Can you move it to .h file?

Reply:
> These 2 APIs can be inline API with single line. Can you move it to .h file?

Yes, will move them to vmcs.h